draft-private-access-tokens.md

```diff
@@ -1080,49 +1092,71 @@ computing the mapping output.
 
 ### Client Behavior {#client-stable-mapping}
 
-Given a client secret (CLIENT_SECRET) and the corresponding key (CLIENT_KEY),
-Clients produce client_origin_index using a Prime Order Groups (POGs) as
-described in {{!VOPRF=I-D.irtf-cfrg-voprf, Section 2.1}}. In particular,
-the functions RandomScalar(), Generator(), SerializeScalar(), SerializeElement(),
-and HashToScalar() are used in the following way.
+This section describes the Client behavior for generating an index request and proof.
+
+#### Index Request {#index-request}
+
+Given a client key (CLIENT_KEY), Clients produce `index_request` using a Prime Order
+Group (POG) as described in {{!VOPRF=I-D.irtf-cfrg-voprf, Section 2.1}}.
+In particular, the functions RandomScalar(), SerializeScalar(), and SerializeElement(),
+are used as follows:
 
 1. Generate a random scalar blind and multiply CLIENT_KEY and the group
    generator by this blind, yielding a blinded key and blinded generator.
-1. Compute a Schnorr proof-of-knowledge demonstrating knowledge of the
-   discrete log of the blinded key with respect to the blinded generator.
-1. Serialize the blinded key, blinded generator, and proof, yielding
-   client_origin_index_key, client_origin_index_base, and client_origin_index_proof,
-   and concatenate each, yielding client_origin_index_request.
-1. Serialize the blind, yielding client_origin_index_blind.
-1. Output client_origin_index_blind, client_origin_index_request.
+1. Serialize the blinded key and blinded generator yielding
+   index_key and index_base, respectively, and concatenate each to produce index_request.
+1. Serialize the blind, yielding index_blind.
+1. Output index_blind, index_request.
 
 In pseudocode, this is as follows:
 
 ~~~
-// Generate blind, blinded key, blinded generator, and the proof
+// Generate blind, blinded key and blinded generator
 blind = RandomScalar()
 blind_key = blind * CLIENT_KEY
-blind_generator = blind * Generator()
-blind_proof = SchnorrProof(CLIENT_SECRET, blind_generator, blind_key)
+blind_generator = ScalarMultBase(blind)
 
 // Serialize and produce outputs
-client_origin_index_proof = SerializeProof(blind_proof)
-client_origin_index_key = SerializeElement(blind_key)
-client_origin_index_base = SerializeElement(blind_generator)
-client_origin_index_request = concat(client_origin_index_key,
-    client_origin_index_base, client_origin_index_proof)
-client_origin_index_blind = SerializeScalar(blind)
+index_key = SerializeElement(blind_key)
+index_base = SerializeElement(blind_generator)
+index_request = concat(index_key, index_base)
+index_blind = SerializeScalar(blind)
+~~~
+
+### Client Index Proof {#index-proof}
+
+Given a client secret (CLIENT_SECRET), Clients produce proof of their request
+based on the following inputs defined in {{request-one}}: `token_key_id`,
```

Comment:
another n00b question: what is the purpose of this proof? From the rest of the construction, this is for the Issuer, but I don't understand what this proof is about.

Reply:
See comment above =)